Small change to note `<Link />` executes server methods (#41577)
vercel/next.js: docs/routing/introduction.md

- `/about` → `pages/about.js`
- `/blog/hello-world` → `pages/blog/[slug].js`

> Any `<Link />` in the viewport (initially or through scroll) will be prefetched by default (including the corresponding data) for pages using [Static Generation](/docs/basic-features/data-fetching/get-static-props.md). The corresponding data for [server-rendered](/docs/basic-features/data-fetching/get-server-side-props.md) routes is prefetched _only when_ the <Link /> is clicked.

Reviewer: ```suggestion
Any `<Link />` in the viewport (initially or through scroll) will be prefetched by default (including the corresponding data) for pages using [Static Generation](/docs/basic-features/data-fetching/get-static-props.md). The corresponding data for [server-rendered](/docs/basic-features/data-fetching/get-server-side-props.md) routes is fetched _only when_ the <Link /> is clicked.
```
It's not prefetching if the link has already been clicked it is just fetching the data as needed for the transition since it is actually in progress.

> Any `<Link />` in the viewport (initially or through scroll) will be prefetched by default (including the corresponding data) for pages using [Static Generation](/docs/basic-features/data-fetching/get-static-props.md). The corresponding data for [server-rendered](/docs/basic-features/data-fetching/get-server-side-props.md) routes is _not_ prefetched.

Reviewer 2: ```suggestion
Any `<Link />` in the viewport (initially or through scroll) will be prefetched by default (including the corresponding data) for pages using [Static Generation](/docs/basic-features/data-fetching/get-static-props.md). The corresponding data for [server-rendered](/docs/basic-features/data-fetching/get-server-side-props.md) routes are prefetched once the <Link /> is clicked.
```

> Also note that when navigating to another page using `<Link />` even tough routing happens client side, the data for that page (via `GetServerSideProps`) _is_ fetched before navigating to the new route.

Reviewer 3: Lets just add it to line 71 at the end.

> routes is _not_ prefetched, but it will be once the `<Link />` is clicked.

Author: I prefer to be more explicit (your version reads a little bit more complicated to me as a non-native speaker), but that version also works for me.

Reviewer 2: See https://github.com/vercel/next.js/pull/41577#discussion_r999992319

> Also note that when navigating to another page using `<Link />` even tough routing happens client side, the data for that page (via `GetServerSideProps`) _is_ fetched before navigating to the new route.
> 

Reviewer 2: ```suggestion
```

I reworded and applied the suggestion by @styfle, thanks for making us aware! The existing sentence explains well enough how prefetching behaves, with a link to server-rendering, so it's a better fit to just expand that sentence IMO, too.
